.github/workflows/stale-issues.yml

```
@@ -0,0 +1,22 @@
+name: Close inactive issues
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          ascending: false
+          operations-per-run: 300
+          days-before-issue-stale: 30
+          days-before-issue-close: 7
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 30 days with no activity. If there is no activity in the next 7 days, the issue will be closed."
+          close-issue-message: "This issue was closed because it has been inactive for 7 days since being marked as stale. Please open a new issue if you believe you are encountering a related problem."
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          enable-statistics: true
```

```suggestion
          enable-statistics: true
          exempt-issue-labels: 'needs review'
```
I'd like to close stale issues, but I don't think issues should be closed that haven't even been triaged. So I'd like to exclude the 'needs review' label.